Source/WebCore/loader/ResourceLoadInfo.h

```diff
@@ -110,6 +112,7 @@ struct ResourceLoadInfo {
     OptionSet<ResourceType> type;
     bool mainFrameContext { false };
     RequestMethod requestMethod { RequestMethod::None };
+    Vector<URL> ancestorFrameURLs { };
```

Comment:
`Vector` is empty by default, no need for `{ }`.
```suggestion
    Vector<URL> ancestorFrameURLs;
```